sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -0,0 +1,33 @@
+"""Main file to store schema definition"""
+from sqlalchemy import (
+    Boolean,
+    Column,
+    DateTime,
+    Integer,
+)
+from sqlalchemy.orm import DeclarativeBase
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class UniversalSpinTable(Base):
+    """Universal Spin Table schema"""
+
+    __tablename__ = "universal_spin_table"
+    id = Column(Integer, primary_key=True)
+    spin_number = Column(Integer, primary_key=True)
+    spin_start_sc_time = Column(Integer)
+    spin_start_utc_time = Column(DateTime(timezone=True))
+    star_tracker_flag = Column(Boolean)
+    spin_duration = Column(Integer)
+    thruster_firing_event = Column(Boolean)
+    repointing = Column(Boolean)
+    repointing_number = Column(Integer)
```

Comment:
Can the value for these ever be `NULL`? For any ones that cannot be `NULL`, you could add the `nullable=False` parameter.

Reply:
Should this be a foreign key? Like we want a repointing table with information about repoints too? When a repoint started/stopped for example.

Reply:
Yeah, I think all of the field should have a value and shouldn't be null. But we can change that later if things changes. For now, I will set them to not be null.

About foreign key, I have not thought about it. That's a good point though.

Reply:
@greglucas can you revisit foreign key in upcoming PR because I like to think about that little bit.

Reply:
Yes, no problem at all! Maybe add a quick comment/TODO above those lines so we don't completely forget about it.